chore(doc): explain ComponentWithFormTrait getDataModelValue override (#986)
symfony/ux: src/LiveComponent/doc/index.rst

    <input data-model="norender|max">

> You might as well deref for all fields ``*`` of the form:

Reviewer: ```suggestion
You can also defer re-rendering all fields for the form using ``*``:
```

Reviewer: Maybe let's move this section down to the end as below talked about the max field from the example above.

Author: Agreed, will rework this

Author: I've used your sentence, but did not understand `move this section down to the end`
after which word you want it please?

.. code-block:: html+twig

>     {{ form_start(form, {attr: {'data-model': 'norender|*'}}) }}

Reviewer 2: You can also override this method - https://github.com/symfony/ux/blob/2.x/src/LiveComponent/src/ComponentWithFormTrait.php#L226-L229 - that's how I imagined people would do this :)

    <input data-model="norender|max">

> You can also defer re-rendering all fields for the form using ``*``:

Reviewer 2: ```rst
For a form using ``ComponentWithFormTrait``, override the ``getDataModelValue()`` method::

private function getDataModelValue()
{
    return 'norender|*';
}
```

    <input data-model="norender|max">

For a form using ``ComponentWithFormTrait``, override the ``getDataModelValue()`` method::

private function getDataModelValue()
{
    return 'norender|*';

> }

Reviewer 2: Indent all of this code (sorry if I confused with my comment)

.. tip::

    You can also define this value inside Twig::

> .. code-block:: html+twig

Reviewer 2: The code block should be indented so that it's inside of the `tip`. And then the actual code below should be indented 4 more.
